install_abba.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+scriptpath=$(realpath $(dirname $0))
+source "$scriptpath/global_function.sh"
```

Comment:
On peut réduire le code ainsi ? qu'en penses tu ?
```suggestion
if [[ "$OSTYPE" == "linux-gnu"* ]] || [[ "$OSTYPE" == "msys" ]]; then
```

Reply:
Alors je ne suis pas sûr de bien comprendre la modif avec le commentaire, mais si c'est plus compact et que ça évite des duplicats de code, oui, impecc! Tu veux ouvrir un PR ? Peut etre que cela sera plus simple pour moi de comprendre la modif.

Reply:
Yep pas de soucis je la fait partout et te fait une PR je t'avoue que je découvre ce genre de fonctionnalité de VSCode